src/clj/harja/kyselyt/tehtavamaarat.sql

```diff
@@ -74,7 +74,7 @@ from toimenpideinstanssi tpi
                  on toteumat.toimenpidekoodi = tpk.id
                    and toteumat.urakka_id = tpi.urakka
        join tehtavaryhma tr on tpk.tehtavaryhma = tr.id
-where tpi.urakka in (select id from urakat)
+where tpi.urakka in (select id from urakat) AND tpk.yksikko NOT ilike 'euro%'
```

Comment:
Tiedoksi vain, että tämä tulee konfliktaamaan, kunhan mergeän (voi olla että ehdit ensin :) ) sen mun korjauksen noihin järjestyksiin. 
Huomasin nimittäin, että tässä haetaan sekä urakka taulu, että halutut urakat tuolla "with as" hommassa.
Ja tuplana ei kannata urakoita hakea. Niinpä muutin sen niin, että "join urakka u" muuttuu muotoon "urakat u", koska se jo sisältää kaikki tarvittavat tiedot urakoista.

Reply:
okei. oli muutenki minusta lähtökohtaisesti huono idea jakaa saman raportin minitaskit monelle eri ihmiselle.

Reply:
sovittiimn että siirretään markuksen pulleroon nämä tän PR:n kaksi riviä